src/main/java/org/broadinstitute/hellbender/utils/GATKProtectedMathUtils.java

```diff
@@ -157,13 +159,101 @@ public static double stdDev(final double ... values) {
         return Math.sqrt(new Variance().evaluate(values));
     }
 
+    /**
+     * Create a set number of points, linearly spaced, between a minimum and maximum.
+     *
+     * http://stackoverflow.com/questions/6208878/java-version-of-matlabs-linspace
+     *
+     * NaN are allowed, but will likely give useless results.
+     *
+     * @param min starting value
+     * @param max ending value
+     * @param points number of points, must be greater than -1
+     * @return Never {@code null}
+     */
+    public static double[] linspace(final double min, final double max, int points) {
```

Comment:
I'm not a fan of the MATLAB name -- `linspace` and `seq` (R) are only clear if they're built-in language features used so frequently that everyone can memorize the short names, which they are in those languages.  But for our purposes this should be a more descriptive (and camel case) name.  Like `arithmeticSequence` or `evenlySpacedPoints` or whatever.